source/_integrations/xiaomi_miio.markdown

```diff
@@ -906,35 +906,12 @@ Currently, the supported features are:
   - battery
   - time_stat
 
-Please follow the instructions on [Retrieving the Access Token](/integrations/xiaomi_miio/#retrieving-the-access-token) to get the API token.
-
 ### Configuration
 
-To add a Xiaomi Mi Air Quality Monitor to your installation, add the following to your `configuration.yaml` file:
+Please follow the instructions on [Retrieving the Access Token](/integrations/xiaomi_miio/#retrieving-the-access-token) to get the API token to use during configuration flow setup.
 
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: xiaomi_miio
-    host: IP_ADDRESS
-    token: YOUR_TOKEN
-```
+To add a Xiaomi Mi Air Quality Monitor to your installation, click Configuration in the sidebar, then click Integrations and then click the + icon in the lower right and find xiaomi_miio. You will then be presented with a form in which you will need to fill in the “IP address” and 32 characters “token”. After you click submit, you will have the opportunity to select the area that your devices are located.
```

Comment:
It is not possible to use `{% include integrations/config_flow.md %}`, the header level is wrong. 
You could copy the content of `source/_include/integrations/config_flow.md` and modify the header.
I don't know if there is a better way to do this.

Reply:
> It is not possible to use {% include integrations/config_flow.md %}, the header level is wrong.

That is not possible even for other reasons, as that assumes the full integration to be a configuration flow. In previous PRs it has been established a single config section can be added to the top once the full integration is migrated.